middle_end/flambda/from_lambda/closure_conversion.ml

```diff
@@ -1063,12 +1063,15 @@ let ilambda_to_flambda ~backend ~module_ident ~module_block_size_in_words
       ~filename (ilam : Ilambda.program) =
   let module Backend = (val backend : Flambda_backend_intf.S) in
   let compilation_unit = Compilation_unit.get_current_exn () in
+  let bound_error_symbol =
+    Lambda_to_flambda_primitives_helpers.caml_ml_array_bound_error
+  in
   let t =
     { backend;
       current_unit_id = Compilation_unit.get_persistent_ident compilation_unit;
       symbol_for_global' = Backend.symbol_for_global';
       filename;
-      imported_symbols = Symbol.Set.empty;
+      imported_symbols = Symbol.Set.singleton bound_error_symbol;
```

Comment:
Could you leave a CR saying that ideally, externals shouldn't be part of the imported symbols ?

Reply:
done